README.md

```diff
@@ -1 +1,11 @@
-# nb2
\ No newline at end of file
+# nb2
+
+An opinionated framework for network function development. Written in Rust, inspired by [NetBricks](http://netbricks.io/) and built on Intel's [Data Plane Development Kit](https://www.dpdk.org/).
+
+# Overview
+
+The project is currently undergoing heavy development. The goal is to offer an ergonomic framework for network function development that traditionally has high barriers of entry for engineers. We want a tool to efficiently manipulate network packets that's type safe, memory safe and thread safe.
+
+# Quickstart
+
+We are working on better documentation and a "Getting started" guide. For now, if you are interested in trying out the pre-release bits, head over to our [docker images](https://github.com/williamofockham/utils) repo and follow the directions to create a local development environment with DPDK and nb2. Then dig around the examples to see for yourself how easy it is to write a program that can filter, parse and create network packets.
```

Comment:
Remove Then, replace with `Afterward, dig around the examples and see for yourself` 

Also, `filter, parse, and...`